Azure/communication-ui-library-android
# [Chat][Feature] send read receipt

## Purpose
Add presentation layer logic for sending read receipt for the latest message on the bottom of chat screen.

## Does this introduce a breaking change?


## Pull Request Type
What kind of change does this Pull Request introduce?



## Pull Request Checklist




## How to Test
Join chat and use other client to send some message into chat thread to see if the message on the other client being marked as read properly.
